node/network/statement-distribution/src/lib.rs

```diff
@@ -1466,14 +1466,18 @@ async fn handle_network_update(
 					maybe_authority: maybe_authority.clone(),
 				},
 			);
-			if let Some(authority) = maybe_authority {
-				authorities.insert(authority, peer);
+			if let Some(authority_ids) = maybe_authority {
+				authority_ids.into_iter().for_each(|a| {
+					authorities.insert(a, peer);
```

Comment:
The code here and the signature of `authorities: &mut HashMap<AuthorityDiscoveryId, PeerId>` still assumes one `PeerId` per `AuthorityDiscoveryId`. You've tweaked that in collator_side and gossip-support (only used for logs), should we also change it here?

Reply:
Actually, it's not critical here, only used for prioritizing propagation to our group (priority_peers).